src/ios/VLCPlugin.m

```diff
@@ -8,6 +8,7 @@
 #import "CDVReachability.h"
 #import "VLCPlugin.h"
 #import "DDLog.h"
+#import <MWFeedParser/NSString+HTML.h>
```

Comment:
This was added to clean the HTML tags from the title property for the NYPRStream